recommenders/models/rbm/rbm.py

```diff
@@ -691,3 +699,27 @@ def predict(self, x):
         vp = self.sess.run(v_, feed_dict={self.vu: x})
 
         return vp
+
+    def save(self, dir_name):
```

Comment:
minor change, to give more flexibility to the user we can add a model name parameter:
```
def save(self, dir_name, model_name=MODEL_CHECKPOINT):
```

Reply:
Added a `file_path` parameter instead of the two. If the parent directory in the file path does not exist, it will be created.